samples/snippets/list_training_phrases_test.py

```diff
@@ -14,12 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import google.auth
+import os
 
 import list_training_phrases
 
 
-_, PROJECT_ID = google.auth.default()
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
```

Comment:
The existing code to fetch the project works in more environments. If `GOOGLE_CLOUD_PROJECT` is set it will be respected by the auth library and override any other detected projects, see [code](https://github.com/googleapis/google-auth-library-python/blob/e756f08dc78616040ab8fbd7db20903137ccf0c7/google/auth/_default.py#L448-L450)

Reply:
SGTM. I've reverted this change.